templates/dns.zone

```diff
@@ -0,0 +1,10 @@
+; Cluster ${cluster_id}
+$ORIGIN ${node_name_suffix}.
+
+api       IN A     ${api_vip}
+api-int   IN A     ${internal_vip}
+*.apps    IN A     ${router_vip}
```

Comment:
It's not clear yet that we want this DNS structure for APPUiO Managed Rancher, we'll need to figure out what RKE actually needs and what we can define freely.